### ISSUE TITLE
Build failure: GHC.Prim does not export Any

### ISSUE THREAD

**Issue by DanBurton** [louispan/data-distinct#2]
Build failure with GHC 8.2.1 as seen on the Stackage build server:

```
[ 7 of 13] Compiling Data.Diverse.Many.Internal ( src/Data/Diverse/Many/Internal.hs, dist/build/Data/Diverse/Many/Internal.o )

src/Data/Diverse/Many/Internal.hs:100:18: error:
    Module ‘GHC.Prim’ does not export ‘Any’
    |
100 | import GHC.Prim (Any, coerce)
    |                  ^^^

```

**Comment by louispan** [comment]
Thanks, I've uploaded a new version on hackage which imports Any from GHC.Exts

**Comment by louispan** [comment]
Fixed in 7ed6c517f1f46f397bfd37360900fb45c1b164c0
